pandas/io/formats/style.py

```diff
@@ -3885,18 +3898,20 @@ def css_calc(x, left: float, right: float, align: str, color: str | list | tuple
     rgbas = None
     if cmap is not None:
         # use the matplotlib colormap input
-        with _mpl(Styler.bar) as (plt, mpl):
+        with _mpl(Styler.bar) as (_pyplot, _matplotlib):
```

Comment:
looks like the first one `_pyplot` is unused - would it work to just set it to `_`?